### PR TITLE
[android] fix gradle build setup for explorer, bump to gradle 2.2.0

### PR DESCRIPTION
**Pull request by gilbox**
fix the directories. 

@spikebrehm @lelandrichardson @felipecsl

### CODE REVIEW

File: android/build.gradle [hunk 1]
```diff
@@ -2,23 +2,22 @@
...
-          // to be installed at `./android/node_modules`.
-          url "$projectDir/../node_modules/react-native/android"
-      }
       maven {
           // For developing the example app, expect this library to be installed as a node module
           // inside of the example app. So traverse from `./android/example/node_modules/react-native-maps/android`
           // to `./android/example/node_modules/react-native/android`.
           // react-native should be installed since it's a peer dependency
-          url "$projectDir/../../react-native/android"
+          url "$projectDir/../example/node_modules/react-native/android"
```
Comment (gilbox) [inline android/build.gradle]: @spikebrehm the comment didn't make any sense. I updated it.

Comment (gilbox) [inline android/build.gradle]: @spikebrehm I had mixed up commits in different branches. it's fixed now.

File: android/build.gradle [hunk 2]
```diff
@@ -9,14 +9,11 @@ buildscript {
...
           // For developing the library outside the context of the example app, expect `react-native`
-          // to be installed at `./android/node_modules`.
+          // to be installed at `./node_modules`.
           url "$projectDir/../node_modules/react-native/android"
       }
       maven {
-          // For developing the example app, expect this library to be installed as a node module
-          // inside of the example app. So traverse from `./android/example/node_modules/react-native-maps/android`
-          // to `./android/example/node_modules/react-native/android`.
-          // react-native should be installed since it's a peer dependency
```
Comment (spikebrehm) [inline android/build.gradle]: Isn't this a useful comment?

**Comment by spikebrehm** [comment]
Do we need to release a patch fix to 0.9 for this as well?

File: android/build.gradle (hunk 2 above)
Comment (gilbox) [inline android/build.gradle]: It's completely wrong, though. We don't pull from `node_modules` for the example app

**Comment by gilbox** [comment]
@spikebrehm afaik it really only effects the explorer.

**Pull request merged**